docs/quickstarts/nextjs.mdx

```diff
@@ -87,6 +91,8 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
       children: React.ReactNode
     }) {
       return (
+        // Active session and user context
+        // available throughout your app

```

Comment:
These are the most common things that people pull from provider, but there are a lot of other things you can pull from it as well, so I'm not sure we'd want to add a comment like this rather than maybe like a link to clerk provider docs somewhere

Reply:
yeah I agree, no need for the comment. the highlighting of the lines plus the copy is enough. 
and we have a link to clerkprovider in the copy above this code example